pkg/ovs/ovsctl/appctl.go

```diff
@@ -227,3 +228,35 @@ func (c *ovsCtlClient) GetDPFeatures() (map[DPFeature]bool, error) {
 	}
 	return features, nil
 }
+
+// DeleteDPInterface deletes OVS datapath interface, and it returns with no error if the interface does not exist.
+func (c *ovsCtlClient) DeleteDPInterface(name string) error {
+	cmd := fmt.Sprintf("dpctl/show ovs-system")
+	out, execErr := c.runAppCtl(cmd, true)
+	if execErr != nil {
+		return execErr
+	}
+	scanner := bufio.NewScanner(strings.NewReader(string(out)))
+	scanner.Split(bufio.ScanLines)
+	for scanner.Scan() {
+		line := scanner.Text()
+		fields := strings.Split(line, ": ")
+		nameStr := fields[1]
```

Comment:
does it work with the first line `system@ovs-system:`? It panics according to my test. https://go.dev/play/p/Ow94O1eg6g0

Reply:
Thanks for pointing out, done